README.md

```diff
@@ -1,2 +1,202 @@
-# saritasa-github-actions
-GitHub actions
+## CI-wordpress reusable workflow
+
+### Deploying
```

Comment:
add links to proper yaml files so we can open them right away.

Reply:
done

Reply:
But below the text, in the examples there are also links to files and listing examples.